lib/mix/lib/mix/tasks/new.ex

```diff
@@ -222,6 +222,9 @@ defmodule Mix.Tasks.New do
         [applications: [:<%= @app %>]]
       end
       ```
+
+  If [published on Hexdocs](https://hex.pm/docs/tasks#hex_docs), the docs can
```

Comment:
I just looked through the official texts we have because I didn't really know. It seems like we use "Hexdocs" exclusively. We could change it to "HexDocs" or maybe "Hex docs" if that looks better though.